BootloaderCommonPkg/Library/UsbBusLib/UsbIoPeim.c

```diff
@@ -194,6 +195,14 @@ PeiUsbBulkTransfer (
     return EFI_INVALID_PARAMETER;
   }
 
+  if ((PeiUsbDev->EndpointDesc[EndpointIndex]->Attributes & USB_ENDPOINT_TYPE_MASK) == USB_ENDPOINT_INTERRUPT) {
+    // Since USB HC PPI has no interrupt transfer support, reuse bulk transfer intead.
```

Comment:
a typo in comments: intead should be instead.

Reply:
will fix it.

Reply:
updated the comments.